### ISSUE TITLE
Attempting to configure pageSummaryMetrics results in duplicate entries?

### ISSUE THREAD

**Issue by grantyoung** [sitespeedio/sitespeed.io#3111]
[While trying to configure what metrics are shown on the Page Metrics page](https://www.sitespeed.io/documentation/sitespeed.io/configure-html/) it resulted in some entries still remaining leading to duplicate entries:

![Screenshot 2020-07-31 at 10 16 29](https://user-images.githubusercontent.com/52497556/89020495-eac8f080-d316-11ea-88fc-dbc0cc0c6b09.png)

Config used (in json, `14.2.1`):

```
{
  "html": {
    "pageSummaryMetrics": [
      "timings.firstContentfulPaint",
      "timings.largestContentfulPaint",
      "cpu.totalBlockingTime",
      "timings.SpeedIndex",
      "timings.LastVisualChange",
      "transferSize.total",
      "score.performance"
    ]
}

It appear the first 3 entries in the table - First Visual Change, Speed Index and Last Visual Change - are permanent?

**Comment by soulgalore** [comment]
I can reproduce it with: 

`docker run --rm -v "$(pwd)":/sitespeed.io sitespeedio/sitespeed.io https://www.sitespeed.io/ -n 1 --config config.json`

I think somewhere in the code we add some Visual Metrics independently if the use chose them or not.

**Comment by soulgalore** [comment]
I released the fix in 14.2.3, let me know if it works for you now!

**Comment by grantyoung** [comment]
> I released the fix in 14.2.3, let me know if it works for you now!

Apologies for the delay in getting back to you. The issue is now indeed fixed! Many thanks!
